### ISSUE TITLE
[DevHub] - Capabilities section of the register_post_type function reference has some incorrect capabilies

### ISSUE THREAD

**Issue by jonathanbossenger** [WordPress/Documentation-Issue-Tracker#729]
## Issue Description

The Parameter detail section of the capabilities parameter describes the different capabilities mapped to the custom post-type object as follows:

3 meta capabilities:
- edit_post, read_post, and delete_post

4 primitive capabilities
 - edit_posts
 - edit_others_posts
 - publish_posts
 - read_private_posts

8 primitive capabilities used in map_meta_cap()
 - read
 - delete_posts
 - delete_private_posts
 - delete_published_posts
 - delete_others_posts
 - edit_private_posts
 - edit_published_posts
 - create_posts

However, this is not 100% correct. 

- delete_posts should be listed under the first set of primitive capabilities, not the primitive capabilities used in map_meta_cap()
- create_posts should not be included in the primitive capabilities used in map_meta_cap

In the description, the number of capacities mapped should also be updated to reflect these changes.

The correct list of capabilities can be found on the [get_post_type_capabilities](https://developer.wordpress.org/reference/functions/get_post_type_capabilities/) function reference

## URL of the Page with the Issue

https://developer.wordpress.org/reference/functions/register_post_type/#capabilities

## Section of Page with the issue

The section on the capabilities argument

## Why is this a problem?

It creates confusion for the developer. For example, they might think they map the delete_posts capability, when it's already automatically mapped. It also creates distrust in the rest of the documentation, which is already slightly confusing when learning about how capabilities are mapped for a custom post type.

## Suggested Fix

Update the capabilities to match those from the [get_post_type_capabilities](https://developer.wordpress.org/reference/functions/get_post_type_capabilities/) function reference. Also, include a note on create_posts, and that it's automatically mapped to edit_posts by default. 

At the same time, it might be a good idea to use logical breaks in the paragraphs to help the reader consume the content in chunks.

**Comment by zzap** [comment]
Heads up @WordPress/docs-issues-coordinators, we have a new issue open. Time to use 'em labels.

**Comment by jonathanbossenger** [comment]
I also want to add that I'm more than happy to make the suggested changes, I'm just not sure what access I need to do so.

**Comment by zzap** [comment]
@jonathanbossenger I gave you access to edit explanations. It's done in the dashboard and uses a classic editor.

![edit-explanation](https://user-images.githubusercontent.com/1508963/232068369-2d47b68a-7f71-41c8-b982-2d2508a177d9.png)

![explanation-editor](https://user-images.githubusercontent.com/1508963/232068492-168ec1b5-4e0d-4e36-a5cb-137b0a8699de.png)

Let me know if you can't access it and/or need any help.

**Comment by jonathanbossenger** [comment]
@zzap I don't seem to be able to edit the explanation, I only have the New button in my toolbar on that page.

![Uploading register_post_type-Function-WordPress-Developer-Resources (1).jpg…]()

**Comment by stevenlinx** [comment]
I've revised the content:

1.) I've relocated `delete_posts` to the first set of primitive capabilities.

2.) I've deleted `create_posts` under capabilities for `map_meta_cap()`, then include a note saying that it's automatically mapped to `edit_posts` by default.

3.) I've updated the number of capacities mapped to reflect these changes (8 for the first set; 6 for the second set).

**Comment by jonathanbossenger** [comment]
Thanks, @stevenlinx 

At the same time, are you able to check why I'm not able to edit these explanations?

**Comment by stevenlinx** [comment]
@jonathanbossenger

Assuming the following is your WordPress.org profile, I wonder if you routinely check the email tied to this account?
https://profiles.wordpress.org/psykro/

When you check the email, check around the dates of Apr. 14th for any email from WordPress.org. If not, check also the junk email folder (WordPress.org email servers deliver well to other email hosts, so it's unlikely an email ends up in junk email folder, but just in case).

I'm guessing the likely cause is that you've not confirmed the role, especially given what you wrote recently:
"...having dropped the use of the psykro online nickname, which now only exists as my WordPress profile name."

**Comment by jonathanbossenger** [comment]
Hey @stevenlinx I do indeed check that email regularly, as it's my personal account that I check every day 😄 I've checked both my inbox, all mail and spam, and there's nothing on the 14th. The last email I received from any wordpress.org account was the [WordPress.org Documentation] Joining confirmation email, which I accepted. 

@zzap are you able to check that the role update was granted?

**Comment by stevenlinx** [comment]
I think I know what the cause of the issue is.

1.)
> I do indeed check that email regularly ...

This means this is not a missed-to-read-email problem.

> The last email I received from any wordpress.org account was the

This means you do get emails from WordPress.org fine.

> [WordPress.org Documentation] Joining confirmation email, which I accepted.

The role in question is "Explanation Editor" in DevHub, so the confirmation email title should look like:
`[WordPress Developer Resources] Joining confirmation`

2.)
For WP Docs related portals, there are 3 sub-sites. Each sub-site has its own admin dashboard and the users/roles in each are independent.

Make WordPress Documentation
https://make.wordpress.org/docs/

WordPress.org Documentation  (HelpHub)
https://wordpress.org/documentation/

WordPress Developer Resources (DevHub)
https://developer.wordpress.org

You're supposed to be granted a role in DevHub, but was instead granted a role in HelpHub.

As I checked in the admin dashboard of DevHub, you don't have the DevHub role in question.

You have "Handbook Editor" instead of "Explanation Editor".

3.)
I've changed the role to "Explanation Editor".

Let me know how it goes (and make sure you check for any emails).
